src/components/ExpenseForm.js

```diff
@@ -1,21 +1,29 @@
 import { useState } from 'react';
-import { useRef } from 'react';
+import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
 
 export default function ExpenseForm({ addToExpenses }) {
-  const [inputValue, setInputValue] = useState({ title: '', amount: '' });
-  const inputRef = useRef(null);
+  const [inputValue, setInputValue] = useState({
+    title: '',
+    description: '',
+    amount: '',
+    date: new Date().toISOString().split('T')[0],
+  });
+  const navigate = useNavigate();
   const handleSubmit = e => {
     e.preventDefault();
+
     addToExpenses(prevExpenses => [
       ...prevExpenses,
       {
         title: inputValue.title,
         amount: inputValue.amount,
+        description: inputValue.description,
+        date: inputValue.date,
       },
     ]);
-    setInputValue({ title: '', amount: '' });
-    inputRef.current.focus(); //This sets the focus back to the title input
+    console.log('Expense has been added');
```

Comment:
Did you want this as a console.log or as an alert or something for the user?

Reply:
Put it in to check for myself, can probably remove it now, thanks.